controller/internal/enforcer/nfqdatapath/datapath_tcp.go

```diff
@@ -132,6 +134,17 @@ func (d *Datapath) processNetworkTCPPackets(p *packet.Packet) (conn *connection.
 
 	}
 
+	if p.GetTCPFlags()&packet.TCPRstMask != 0 {
```

Comment:
i'm a little concerned about this. i have seen RSTs flying back and forth at the end of connections interleaved with FINs. This I believe needs more testing in other cases.

Reply:
This only catches RSTs during the 3-way handshake.